src/openrct2/interface/Window_internal.h

```diff
@@ -94,6 +94,7 @@ struct rct_window
         const scenario_index_entry* highlighted_scenario;
         uint16_t var_496;
     };
+    uint32_t guestWatchTimer; // Timer for "someone is watching me" thought
```

Comment:
Thus shouldn't become part of this struct it's specific to the peep window. The peep window will need to be refactored to use the new style first